e2e/wdio/platform/tests/feed-input.e2e-spec.ts

```diff
@@ -0,0 +1,161 @@
+import {
+    addValue,
+    clearValue,
+    doesItExist,
+    elementDisplayed,
+    executeScriptAfterTagAttr, executeScriptAfterTagFF,
+    getAttributeByName,
+    getAttributeByNameArr,
+    getCSSPropertyByName,
+    getElementArrayLength,
+    getElementSize,
+    isEnabled,
+    refreshPage,
+    scrollIntoView,
+    sendKeys,
+    setValue, waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { FeedInputPo } from '../pages/feed-input.po';
+import inputPageContent from '../fixtures/appData/feed-input-page-contents';
+import inputPageTestData from '../fixtures/testData/feed-input';
+
+
+describe('Verify Feed Input component', function() {
+    const { feedInputAvatar, feedInputTextArea, feedInput, feedInputNoAvatar, feedInputButton } = new FeedInputPo();
+    const { four_lines_text, eight_lines_text } = inputPageTestData;
+    const { placeholders_array, send_button_tooltip, avatar_tooltip, default_avatar_class } = inputPageContent;
+    const feedInputPage = new FeedInputPo();
+    beforeAll(() => {
+        feedInputPage.open();
+        waitForElDisplayed(feedInputAvatar);
```

Comment:
You already have a waitForElDisplayed with a page element  in the PO method for open(), seems like you shouldn't need it in the beforeAll too.

Reply:
Done.